### PR TITLE
Filled in community assessment for UWaterloo

### PR DESCRIPTION
**Pull request by joanna-chen**

### CODE REVIEW

File: communities/university-of-waterloo/community-assessment.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Resources 
+* since Waterloo is a STEM-focused school and yet, there is not much student-led resources, nor specific resources for those in tech (eg. technical mock interviews, technical panels) 
+* a lot of companies come to Waterloo to recruit and they often hold workshops 
+
+## Accessibility
+* accessibility for the resources are quite open but the downfall is that people often will not seek them out and there just isn't that much to begin with
+ 
+## Interests 
+* Waterloo is msotly an engineering school with a heavy interest in compuster science and software engineering
```
Comment (hackerkid) [inline communities/university-of-waterloo/community-assessment.md]: *msotly mostly

Comment (hackerkid) [inline communities/university-of-waterloo/community-assessment.md]: *compuster computer

File: communities/university-of-waterloo/community-assessment.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Goals
+* because Waterloo has a co-op system, many students are studying to get a good co-op job
+
+## Location 
+* on Waterloo campus would be the best and most convenient 
+* there are many buildings that can accomodate up to thousands of students
```
Comment (hackerkid) [inline communities/university-of-waterloo/community-assessment.md]: *accommodate accommodate

File: communities/university-of-waterloo/community-assessment.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Communication 
+* there is a Facebook group for Computer Science within University of Waterloo
+* if a Github organization was started, it may help for resource sharing
```
Comment (Lionex) [inline communities/university-of-waterloo/community-assessment.md]: What kind of resources do you hope to share?  I've been considering something like this as well, but I'm unsure what kinds of resources are actually useful for the community.  Any thoughts on:

- website template
- guides
    - who to speak to for reserving space
    - how to start an organization
    - what professors support student activities
- a slack-bot to help manage a community
- workshop files

Comment (gillchristian) [inline communities/university-of-waterloo/community-assessment.md]: A Slack bot to help and manage the community, that's an awesome idea!!! @Lionex

**Pull request merged**